artifactory.py

```diff
@@ -701,6 +701,7 @@ def rest_put(
     @staticmethod
     def rest_post(
         url,
+        json_data=None,
```

Comment:
This breaks backwards compatibility

Reply:
json_data with a default value `None`,  Will it also break backward compatibility？
Do you have any suggestions for this？

Reply:
Order of the arguments :)

Reply:
I hava updated it.

Reply:
yes